app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+module SessionsHelper
+  def log_in user
+    session[:user_id] = user.id
+  end
+
+  def current_user
+    if (user_id = session[:user_id])
+      @current_user ||= User.find_by(id: user_id)
+    elsif (user_id = cookies.signed[:user_id])
+      user = User.find_by(id: user_id)
+      if user&.authenticated?(cookies[:remember_token])
+        log_in user
+        @current_user = user
+      end
+    end
+  end
+
+  def check_remember
+    params[:session][:remember_me] == "1" ? remember(user) : forget(user)
```

Comment:
you should research gem figaro to moved "1" to Settings.yml

Reply:
- em không hiểu sao qua chapter sau, quay lại thì một số code ở chapter sau nó lại ở chapter trước. Ví dụ cái này nó ở chapter 9, e sẽ fix ở chapter 9, tí a qua review giúp e. 
- cái gem này là gem config hay gem figaro vậy a, vì Setting.yml nó thuộc gem config mà.